### DISCUSSION TITLE
Anzeige der richtigen Dateninhalte im Gruppenmonitor

### DISCUSSION THREAD

**Discussion by mbrockeu** [brutella/hkknx-public#274] [Ideas]
Ich nutze den Gruppenmonitor gerne zum Debugger weil die ETS Super langsam ist und außerdem auf dem Handy das Teil eh nicht funktioniert.

Leider sind die gelieferten Werte immer in HEX und somit quasi nicht lesbar auf den schnellen Blick. Kann man das ganze irgendwie so machen, dass die Werte gleich korrekt ausgegeben werden, ähnlich wie in der ETS?

**Comment by brutella** [comment]
Wenn in dem importierten ETS Projekt der Gruppenadresse ein Datentyp zugewiesen wurde, dann zeigt hkknx im Gruppenmonitor auch einen lesbaren Wert an. Ist das bei dir der Fall?

**Comment by mbrockeu** [comment]
Naja ich habe einen Datentyp gesetzt in der ETS. Aber der scheint ja nicht genau genug zu sein dass du diesen auswertest.
Oder was erwartest du dort?

<img width="883" alt="image" src="https://github.com/brutella/hkknx-public/assets/16849873/f3e878b1-5162-4c28-9b19-4870de54600b">

**Reply by mbrockeu** [reply]
Habe die Datentypen jetzt mal richtig gesetzt

<img width="831" alt="image" src="https://github.com/brutella/hkknx-public/assets/16849873/57f55c5c-5b33-4da8-bb9d-8ec8abce1961">

kommt aber noch das bei raus:

<img width="1636" alt="image" src="https://github.com/brutella/hkknx-public/assets/16849873/19ba56aa-00aa-4964-9bfc-4b4b71e78830">

Ich vermute mal, du hast auch nicht alle Datentypen hinterlegt?

**Reply by brutella** [reply]
Probier's mal mit der Pre-Release [2.7.0-b2](https://github.com/brutella/hkknx-public/releases/tag/2.7.0-b2). Damit sollte der Datentyp DPT-14-56 (Power/Leistung) angezeigt werden.

Lass es mich wissen, falls weitere Datentypen unterstützt werden sollen.

**Comment by chef76** [comment]
Kann ich bestätigen. Die Leistung zeigt er bei mir im Gruppenmonitor auch nur in HEX an, obwohl der Datentyp gesetzt ist:
![image](https://github.com/brutella/hkknx-public/assets/69462419/1cc92da9-110e-4709-b46c-781e57c9c0d4)

**Comment by d8421** [comment]
Ich habe das selbe Problem bei allen Temperaturen für die Heizungssteuerung. Die Gruppenadresse sind alle als 9.001 Temperatur hinterlegt.  HKKNX sendet hier nur Hex-Werte und liest im Gruppenmonitor auch nur solche.

**Reply by brutella** [reply]
Wird für diese Gruppenadresse im Gruppenmonitor auch der Datentyp "DPST-9-1" angezeigt?

**Reply by d8421** [reply]
Nein, die Spalte bleibt einfach leer im Gruppenmonitor.

**Reply by brutella** [reply]
Das ist im importierten Projekt für diese Gruppenadresse auch kein Datentyp hinterlegt. Hast du den aktuellsten Stand des ETS Projektes in hkknx importiert?

**Reply by d8421** [reply]
Interessant. In der ETS sind bei allen Temperatur-GAen die Daten hinterlegt. Beim Import sind sie dann aber alle leer. Kann ich mir gerade auch nicht erklären; es ist bei allen Temperatur-GA dasselbe. Andere GAen sind nicht davon betroffen.

**Reply by brutella** [reply]
Wenn ich mir das im Detail ansehen soll, kannst du mir dein Projekt per E-Mail an hkknx@hochgatterer.me zukommen lassen.

**Comment by brutella** [comment]
In Version [2.8.0](https://github.com/brutella/hkknx-public/releases/tag/2.8.0) werden viel mehr Dateninhalte korrekt angezeigt als vorher. Gegebenenfalls sollte man das ETS-Projekt nochmals importieren um die richtigen Inhalte zu sehen.
